src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -12,7 +12,8 @@ import AppStore from '../../stores/AppStore';
 import ColorPicker from '../ColorPicker';
 import CustomEventDialog from './Toolbar/CustomEventDialog/CustomEventDialog';
 
-import locations from '../RightPane/SectionTable/static/locations.json';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const locations = require('../RightPane/SectionTable/static/locations.json') as Record<string, string>;
```

Comment:
require will break vite since vite uses es modules; change it to the `import` version